artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/queue/UpdateQueue.java

```diff
@@ -31,7 +31,7 @@
    String name;
 
    @Option(name = "--deleteOnNoConsumers", description = "whether to delete when it's last consumers disconnects)")
-   Boolean deleteOnNoConsumers = null;
+   String deleteOnNoConsumers = null;
```

Comment:
look at Create, allow-anonymous | --require-login

(Remember when I talked about being creative ? :) )

anyways.. I was looking into this, and I found a better way to address this, by using inputs on certain defaults.